### PR TITLE
Bump postgres version to 10.16

### LINKED ISSUE #48: Postgres version bumped to 10.16
At current, the build is failing because the Postgres version is hardcoded at 10.15 and that version is no longer available in the artifact repository.

To fix this in the short-term, we'll increment the pg version to 10.16 - but we should investigate a longer term fix like #21

### PR DESCRIPTION
**Pull request by izgeri**
### What does this PR do?
Bumps pg to 10.16

### What ticket does this PR close?
Resolves #48

#### Change log
- [x] The CHANGELOG has been updated, or
- [ ] This PR does not include user-facing changes and doesn't require a CHANGELOG update

#### Test coverage
- [ ] This PR includes new unit and integration tests to go with the code changes, or
- [x] The changes in this PR do not require tests

#### Documentation
- [ ] This PR does not require updating any documentation, or
- [x] Docs (e.g. `README`s) were updated in this PR, and/or there is a follow-on issue to update docs

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+- Postgres version is incremented to 10.16 from 10.15.
```
Comment (diverdane) [inline CHANGELOG.md:9]: I think this needs a `### Changed` header.

**Review by diverdane — COMMENTED** [review]
One minor change, the rest looks good!

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
```
Comment (izgeri) [inline CHANGELOG.md:8]:
```suggestion

### Changed
```

**Review by diverdane — APPROVED** [review]
LGTM!

**Pull request merged**